main.go

```diff
@@ -21,8 +21,12 @@ func main() {
 }
 
 func do_main() int {
+	var delaySampler = RandomDistributionSampler{RandomConstDistribution{0 * time.Millisecond}}
 	parallelStreams := flag.Int("n", 1, "Number of parallel streams to start immediately")
-	restartDelay := flag.Duration("restartDelay", 500*time.Millisecond, "Time before starting a new stream, when a stream ends (with or without error)")
+	flag.Var(&delaySampler, "restartDelayDistribution", "Define an random distribution for the time before starting a stream."+
+		" This is applied, when streams are initially started and when a stream ends (with or without error). Definition format: "+
+		"<distribution type>:<comma separated list of duration parameters>. Supported distribution types and number of required parameters: "+
+		"'const'(1), 'equal'(2), 'norm'(2). Examples: 'const:500ms', 'const:5s', 'norm:100ms,30ms', 'equal:0ms,1s'.")
```

Comment:
For normal distribution, it is not clear which is the mean, and which is the std-dev...